telegram/constants.py

```diff
@@ -39,6 +39,110 @@
     MAX_MESSAGE_ENTITIES (:obj:`int`): 100 (Beyond this cap telegram will simply ignore further
         formatting styles)
 
+The following constants are attributes of classes:
+
+:class:`telegram.Chat`:
+
+Attributes:
+    CHAT_PRIVATE (:obj:`str`): 'private'
+    CHAT_GROUP (:obj:`str`): 'group'
+    CHAT_SUPERGROUP (:obj:`str`): 'supergroup'
+    CHAT_CHANNEL (:obj:`str`): 'channel'
+
+:class:`telegram.ChatAction`:
+
+Attributes:
+    CHATACTION_FIND_LOCATION (:obj:`str`): 'find_location'
+    CHATACTION_RECORD_AUDIO (:obj:`str`): 'record_audio'
+    CHATACTION_RECORD_VIDEO (:obj:`str`): 'record_video'
+    CHATACTION_RECORD_VIDEO_NOTE (:obj:`str`): 'record_video_note'
+    CHATACTION_TYPING (:obj:`str`): 'typing'
+    CHATACTION_UPLOAD_AUDIO (:obj:`str`): 'upload_audio'
+    CHATACTION_UPLOAD_DOCUMENT (:obj:`str`): 'upload_document'
+    CHATACTION_UPLOAD_PHOTO (:obj:`str`): 'upload_photo'
+    CHATACTION_UPLOAD_VIDEO (:obj:`str`): 'upload_video'
+    CHATACTION_UPLOAD_VIDEO_NOTE (:obj:`str`): 'upload_video_note'
+
+:class:`telegram.ChatMember`:
+
+Attributes:
+    CHATMEMBER_ADMINISTRATOR (:obj:`str`): 'administrator'
+    CHATMEMBER_CREATOR (:obj:`str`): 'creator'
+    CHATMEMBER_KICKED (:obj:`str`): 'kicked'
+    CHATMEMBER_LEFT (:obj:`str`): 'left'
+    CHATMEMBER_MEMBER (:obj:`str`): 'member'
+    CHATMEMBER_RESTRICTED (:obj:`str`): 'restricted'
+
+:class:`telegram.Dice`:
+
+Attributes:
+    DICE_DICE (:obj:`str`): '🎲'
+    DICE_DARTS (:obj:`str`): '🎯'
+    DICE_BASKETBALL (:obj:`str`): '🏀'
+
+:class:`telegram.MessageEntity`:
+
+Attributes:
+    MESSAGEENTITY_MENTION (:obj:`str`): 'mention'
+    MESSAGEENTITY_HASHTAG (:obj:`str`): 'hashtag'
+    MESSAGEENTITY_CASHTAG (:obj:`str`): 'cashtag'
+    MESSAGEENTITY_PHONE_NUMBER (:obj:`str`): 'phone_number'
+    MESSAGEENTITY_BOT_COMMAND (:obj:`str`): 'bot_command'
+    MESSAGEENTITY_URL (:obj:`str`): 'url'
+    MESSAGEENTITY_EMAIL (:obj:`str`): 'email'
+    MESSAGEENTITY_BOLD (:obj:`str`): 'bold'
+    MESSAGEENTITY_ITALIC (:obj:`str`): 'italic'
+    MESSAGEENTITY_CODE (:obj:`str`): 'code'
+    MESSAGEENTITY_PRE (:obj:`str`): 'pre'
+    MESSAGEENTITY_TEXT_LINK (:obj:`str`): 'text_link'
+    MESSAGEENTITY_TEXT_MENTION (:obj:`str`): 'text_mention'
+    MESSAGEENTITY_UNDERLINE (:obj:`str`): 'underline'
+    MESSAGEENTITY_STRIKETHROUGH (:obj:`str`): 'strikethrough'
+
+:class:`telegram.ParseMode`:
+
+Attributes:
+    PARSEMODE_MARKDOWN (:obj:`str`): 'Markdown'
+    PARSEMODE_MARKDOWN_V2 (:obj:`str`): 'MarkdownV2'
+    PARSEMODE_HTML (:obj:`str`): 'HTML'
+
+:class:`telegram.Poll`:
+
+Attributes:
+    POLL_REGULAR (:obj:`str`): 'regular'
+    POLL_QUIZ (:obj:`str`): 'quiz'
+
+
+
+:class:`telegram.ext.ConversationHandler`:
```

Comment:
Upsie :D I didn't think of the fact that we have class variables in `telegram.ext/utils.` Those shouldn't go here, though. Basically, we're just concerned with the constants of TG classes. Please revert :)